src/TemplateWorksheetBuilder.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Staffim\SpreadsheetBuilder;
+
+use PhpOffice\PhpSpreadsheet\Reader;
+use PhpOffice\PhpSpreadsheet\RichText\RichText;
+use PhpOffice\PhpSpreadsheet\Worksheet\Worksheet;
+use PhpOffice\PhpSpreadsheet\Writer;
+
+class TemplateWorksheetBuilder implements WorksheetBuilderInterface
+{
+    protected const START_ROW = 1;
+    protected const START_COLUMN = 'A';
+    protected const END_ROW = 30;
+    protected const END_COLUMN = 'Z';
+
+    /**
+     * @var string
+     */
+    protected $template;
+
+    /**
+     * @var string
+     */
+    protected $templateWorksheetName;
+
+    /**
+     * @var string
+     */
+    protected $startColumn;
+
+    /**
+     * @var string
+     */
+    protected $endColumn;
+
+    /**
+     * @var int
+     */
+    protected $startRow;
+
+    /**
+     * @var int
+     */
+    protected $endRow;
+
+    public function __construct(
+        string $template,
+        string $worksheetName,
+        $startColumn = self::START_COLUMN,
```

Comment:
why can't we use singe `$range` parameter instead of four parameters? There is `PhpOffice\PhpSpreadsheet\Cell\Coordinate` helper for thar calculations

Reply:
fixed